datastore/transaction.go

```diff
@@ -37,6 +37,14 @@ type transactionSettings struct {
 	readOnly bool
 	prevID   []byte // ID of the transaction to retry
 	readTime *timestamppb.Timestamp
+
+	// When set, skips the initial BeginTransaction RPC call to obtain txn id and
+	// uses the piggybacked txn id from first read rpc call.
+	// If there are no read operations on transaction, BeginTransaction RPC call is made
+	// before rollback or commit
+	// Currently, this setting is set but unused
+	// TODO: Use this setting
```

Comment:
Nit: Do we usually add buganizer for TODOs? [go/todo-style](https://goto.google.com/todo-style)

Reply:
Updated